internal/upstreamldap/upstreamldap.go

```diff
@@ -13,10 +13,13 @@ import (
 	"fmt"
 	"net"
 	"net/url"
+	"regexp"
 	"sort"
 	"strings"
 	"time"
 
+	"github.com/google/uuid"
```

Comment:
Unimportant nit: this import is accidentally in a group by itself as grouped by whitespace.

Reply:
Resolved in 05afae60c2d3dd95e805ea4a2899513493a1e31d